var/spack/repos/builtin/packages/py-cleo/package.py

```diff
@@ -12,8 +12,17 @@ class PyCleo(PythonPackage):
     homepage = "https://github.com/sdispater/cleo"
     pypi = "cleo/cleo-0.8.1.tar.gz"
 
-    version("0.8.1", sha256="3d0e22d30117851b45970b6c14aca4ab0b18b1b53c8af57bed13208147e4069f")
+    version("1.0.0a5", sha256="097c9d0e0332fd53cc89fc11eb0a6ba0309e6a3933c08f7b38558555486925d3")
+    version(
+        "0.8.1",
+        sha256="3d0e22d30117851b45970b6c14aca4ab0b18b1b53c8af57bed13208147e4069f",
+        preferred=True,
+    )
 
     depends_on("python@2.7,3.4:3", type=("build", "run"))
+    depends_on("python@3.7:3", when="@1.0.0a5:", type=("build", "run"))
```

Comment:
We should probably just say `@1:`. Once 1.0.0 is officially released, it will likely have these same dependencies